README.md

```diff
@@ -0,0 +1,31 @@
+
+They differentiated species with thresholds for sequence similarity – that is, ITS sequence similarity above 97-98% are indicative of same-species, which is a widely-adopted threshold for species delimitation in Basidiomycota (Nilsson et al. 2008). However, each case was considered individually as this threshold wasn’t suitable for all cases. They also considered if the terminal subclade was well-supported, and whether any overlap existed between intraspecific genetic distance vs. respective interspecific values compared to the closest related taxon (so essentially, more closely related to each other than the next closest clade). 
+
+**Figure 3.** Summary tree of the genus *Ganoderma* inferred from ML analysis, based on ITS sequence data (main dataset, DS; Table 2). Thick lines represent ML bootstrap values (BS) ≥ 65% and Bayesian Posterior Probabilities (BPP) ≥ 0.95. Clades and Clusters within the tree appear as presented in Table 1 and Suppl. material 1: Table S2. Species names correspond to those inferred in this study. Scale bar: 0.01 nucleotide substitutions per site.
+![](https://mycokeys.pensoft.net/showimg.php?filename=oo_481977.jpg)
+
+*Background* 
+![](https://static.inaturalist.org/photos/83806619/small.jpeg?1594492836)
+As an undergraduate, I started a research project to collect and sequence some mushrooms – specifically, of the wood-decay genus *Stereum* – from Iowa. It was difficult to tell the species apart from morphology alone, so I figured I would sequence them and create a phylogenetic tree, using the accepted DNA barcode for fungi, the internal transcribed spacer (ITS). This revealed that one of the most common species in North America, *S. ostrea*, was actually three genetically distinct species that formed a well-supported clade together, and had been previously described but popularly considered synonyms of *S. ostrea* due to their similar morphology. Mapping phenotypic traits on to this phylogeny, however, made it clear that although similar and highly variable, there were a few key characteristics that could be reliably used to differentiate these species. Now, for my master’s thesis work, I want to create a global molecular phylogeny for the genus, as *Stereum* occur commonly worldwide. However, publicly available sequences have issues – a high rate of misidentification, poor sequence quality, and lack of metadata being some of them. Therefore, I will be requesting herbarium samples to examine and sequence, but in the meantime public sequences can still be useful to me in another way.
+
+One thing that is not clear is how many species of *Stereum* there are, how many are described, and how many have yet to be described. By amassing all the ITS sequence data available and creating a program to help me make sense of the data, I can estimate how many species there are and hopefully use metadata to pinpoint where on the globe they can be found, and if I am lucky, additional ecological data such as tree host species. 
```

Comment:
can you expand on what you mean by "creating a program to help me make sense of the data"? What kind of program, to do what, and what kind of sense would you aim at making clear?